src/core_modules/capture-core/metaDataStoreLoaders/programs/quickStoreOperations/storeProgramIndicators.js

```diff
@@ -13,15 +13,15 @@ const convert = (response) => {
         }));
 };
 
-const fieldsParam = 'id,displayName,code,shortName,style,displayInForm,expression,' +
+const fieldsParam = 'id,displayName,code,shortName,style,expression,' +
 'displayDescription,description,filter,program[id]';
 
 export const storeProgramIndicators = async (programIds: Array<string>) => {
     const query = {
         resource: 'programIndicators',
         params: {
             fields: fieldsParam,
-            filter: `program.id:in:[${programIds.join(',')}]`,
+            filter: `displayInForm:eq:true&filter=program.id:in:[${programIds.join(',')}]`,
```

Comment:
To make two separate filter parameters, we need to split the filter into an array with two items, like this:
filter: [ `programid filter`, `display in form filter`],

Reply:
Yes, that worked! :tada: